helm-chart/values.yaml

```diff
@@ -1,152 +1,133 @@
 # Default values for json-rpc-relay.
+affinity: {}
 
-replicaCount: 2
+autoscaling:
+  enabled: true
+  minReplicas: 2
+  maxReplicas: 6
+  targetCPUUtilizationPercentage: 50
+  targetMemoryUtilizationPercentage: 50
+
+config:
+  # This value should be in decimal, not hexadecimal format. Defaulted to testnet
+  CHAIN_ID: '0x128'
+  CLIENT_TRANSPORT_SECURITY: false
+  CONSENSUS_MAX_EXECUTION_TIME: 15000
+  CONTRACT_QUERY_TIMEOUT_RETRIES: 3
+  DEV_MODE: false
+  DEFAULT_RATE_LIMIT: 200
+  ETH_CALL_CACHE_TTL: 200
+  ETH_CALL_DEFAULT_TO_CONSENSUS_NODE: true
+  ETH_GET_LOGS_BLOCK_RANGE_LIMIT: 1000
+  GAS_PRICE_TINY_BAR_BUFFER: 10000000000
+  HBAR_RATE_LIMIT_TINYBAR: 5000000000
+  HBAR_RATE_LIMIT_DURATION: 60000
+  # Use config.hosted.HEDERA_NETWORK when running against a hosted Hedera network.  Valid options are `previewnet`, `testnet`, and `mainnet`
+  hosted:
+    # Defaulted to testnet
+    HEDERA_NETWORK: 'testnet'
```

Comment:
Updated

Reply:
There are some leftover references to config.hosted.HEDERA_NETWORK and config.local.HEDERA_NETWORK in the doc comments of some .yaml files, they should  probably be removed to avoid confusion